src/rust/catcollar/mod.rs

```diff
@@ -86,10 +80,10 @@ use ::std::{
 
 /// Catcollar LibOS
 pub struct CatcollarLibOS {
-    /// Table of queue descriptors.
-    qtable: Rc<RefCell<IoQueueTable>>, // TODO: Move this into runtime module.
+    /// S
```

Comment:
I think some text may have been accidentally deleted here.

Reply:
Yup, thanks! I removed the TODO as well.